distributedcombigrid/src/sgpp/distributedcombigrid/manager/ProcessManager.cpp

```diff
@@ -396,6 +396,33 @@ std::vector<double> ProcessManager::evalErrorOnDFG(const LevelVector& leval, siz
   return g->evalErrorOnDFG(leval);
 }
 
+std::vector<CombiDataType> ProcessManager::interpolateValues(const std::vector<std::vector<real>>& interpolationCoords) {
+  auto numValues = interpolationCoords.size();
+  std::vector<std::vector<CombiDataType>> values (pgroups_.size(), std::vector<CombiDataType>(numValues, std::numeric_limits<double>::quiet_NaN()));
+  std::vector<MPI_Request> requests(pgroups_.size());
+
+  // send interpolation coords as a single array
+  auto coordsSize = numValues* interpolationCoords[0].size();
```

Comment:
haha, moving this part here prevents interpolationCoordsSerial from going out of scope. fixed!